mod-data-export/src/test/resources/firebird/dataexport/features/export-auth-update-headings.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Test Data Export Spring API
+
+  Background:
+    * url baseUrl
+
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+    * configure headers = headersUser
+
+    * def requestBodyOnSuccess = read('classpath:samples/auth_update_headings_success.json')
```

Comment:
Please read file right in the scenario where it's using. 
Because Background running for each scenario. So it reads this 2 files 3 times